pkgs/applications/graphics/apitrace/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "apitrace-${version}";
-  version = "7.1";
+  version = "7.1-git-20170411";
```

Comment:
I've checked the history of `apitrace` releases, it looks like only 2-component versions are used. So the options are:
- `7.2-pre-20170414`
- `7.1-363-ge3509be1`  from `git describe`

I like the second one more. If it's OK, I'll use it and also I'll update the manual.